job-scheduler/src/test/java/org/hawkular/metrics/scheduler/impl/JobSchedulerTest.java

```diff
@@ -112,8 +112,8 @@ protected static void setActiveTimeSlices(DateTime... timeSlices) {
         }
     }
 
-    protected static Set<JobDetails> getScheduledJobs(DateTime timeSlice) {
-        List<JobDetails> jobs = jobsService.findScheduledJobsForTime(timeSlice.toDate(), Schedulers.computation())
+    protected static Set<JobDetailsImpl> getScheduledJobs(DateTime timeSlice) {
```

Comment:
If I use the interface in places where I do not need any specific impl details, I wind up having to do casting which is ugly. I find it easier and cleaner to just use JobDetailsImpl everywhere rather than cast. These are all impl classes any way that are not part of the public API.